ERCS/erc-4377.md

```diff
@@ -0,0 +1,361 @@
+---
+eip:
```

Comment:
```suggestion
eip: 7660
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename and all references to the ERC number.